x-pack/plugins/task_manager/server/integration_tests/managed_configuration.test.ts

```diff
@@ -112,4 +119,28 @@ describe('managed configuration', () => {
     );
     expect(logger.debug).toHaveBeenCalledWith('Task poller now using interval of 3600ms');
   });
+
+  test('should lower max workers when Elasticsearch returns "cannot execute [inline] scripts" error', async () => {
```

Comment:
It looks like the poll interval also increases in response to this error...should we add a test for that?